src/mds/MDSMap.h

```diff
@@ -140,12 +130,15 @@ class MDSMap {
     mds_rank_t standby_for_rank;
     std::string standby_for_name;
     fs_cluster_id_t standby_for_fscid;
+    bool standby_replay;
```

Comment:
I'm concerned we may be inappropriately losing the standby-replay intentions of older daemons; will come back to that...